lib/views/summary.erb

```diff
@@ -3,50 +3,91 @@
+  <div class="col">
+    <div class="container">
+      <div class="row p-5">
+      <div class='col-1'></div>
+        <div class="col-4 rounded pt-2 bg-white" >
+        <h3 style="text-align:center;">Billing Details</h3>
+        <br/>
+        <p> <%= @customer[:billing_address_line1] %> <br/> 
+        <%= @customer[:billing_address_line2] %>  <br/> 
+        <%= @customer[:billing_city] %>  <br/> 
+        <%= @customer[:billing_county] %>  <br/> 
+        <%= @customer[:billing_postcode] %>  <br/> <br/>
+        <%= @customer[:billing_phone_number] %>  <br/> 
+        <%= @customer[:billing_email_address] %> </p>
+        </div>
+        <div class='col-1 border-right '></div>
+        <div class='col-1'></div>
+        <div class="col-4 pt-2 bg-white rounded" >
+          <h3 style="text-align:center;">Shipping Details</h3>
+        <br/>
+        <p> <%= @customer[:shipping_address_line1] %> <br/> 
+        <%= @customer[:shipping_address_line2] %>  <br/> 
+        <%= @customer[:shipping_city] %>  <br/> 
+        <%= @customer[:shipping_county] %>  <br/> 
+        <%= @customer[:shipping_postcode] %>  <br/> <br/>
+        <%= @customer[:shipping_phone_number] %>  <br/> 
+        <%= @customer[:shipping_email_address] %> </p>
         </div>
+        <div class='col-1'></div>
       </div>
       <div class="row">
-        <div class="col">
+        <div class="col" >
 
- <table class="table table-striped">
-  <thead>
-    <tr>
-      <th scope="col">ID</th>
-      <th scope="col">Name</th>
-      <th scope="col">Price</th>
-      <th scope="col">Quantity</th>
-    </tr>
-  </thead>
-  <tbody>
-    <% @items.each do |item| %>
-    <tr>
-      <td><%= item[:part_id] %></td>
-      <td><%= item[:part_name] %></td>
-      <td>£ <%= item[:part_price] %></td>
-      <td> <span style="font-size: 15px; font-weight: normal;" class="badge badge-primary badge-pill"> <%= item[:part_quantity] %> </span>
-  </td>
-    </tr>
-     <% end %>
-  </tbody>
-</table>
-      </div>
+      <table class="table table-hover border rounded bg-white">
+        <thead>
+          <tr>
+            <th scope="col">ID</th>
+            <th scope="col">Name</th>
+            <th scope="col">Price</th>
+            <th scope="col">Quantity</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @items.each do |item| %>
+          <tr>
+            <td style="width: 10%;"><%= item[:part_id] %></td>
+            <td style="width: 40%;" ><%= item[:part_name] %></td>
+            <td style="width: 40%;" ><%= item[:part_price] %></td>
+            <td style="width: 10%;" > <span style="font-size: 15px; font-weight: normal;" class="badge badge-primary badge-pill"> <%= item[:part_quantity] %> </span>
+        </td>
+          </tr>
+          <% end %>
+        </tbody>
+      </table>
+            </div>
+          </div>
     </div>
     </div>
+  </div>
+  <div class="row">
+  <div class="col p-4 pb-5">
+  <form method="GET" action="/add-items">
```

Comment:
This could just be an `a`

`<a class="btn btn-secondary ml-3" href="/add-items">Back</a>` (classes might be slightly off, don't know bootstrap 4 that well)